### ISSUE TITLE
Thompson's "Regular expression search" generated IBM 7094 code, not PDP-11

### ISSUE THREAD

**Issue by nsajko** [ethanpailes/thesis#1]
Hi, I like the Skip Regex idea, so I was glad to find that your thesis is freely available here (couldn't be accessed from Google Scholar). Is it OK to put comments and questions about the thesis here?

I want to point out a small error in the _Some Regex History_ section: the program in Thompson's 1968 paper does not generate PDP-11 machine code as you wrote, rather it generates instructions (assembly, I think) for the older [IBM 7094](https://en.wikipedia.org/wiki/IBM_7090#IBM_7094) mainframes.

**Comment by ethanpailes** [comment]
Yeah, feel free to leave comments here. I'll have to take another look at the paper. I think I may have seen some other secondary source that gave me the impression it generated PDP-11 code.
